components/export_table/ExportTableButton.vue

```diff
@@ -1,8 +1,98 @@
 <script setup lang="ts">
+import type {Teacher} from "~/model/teacher"
+import type {DropdownItem} from "#ui/types"
+import {useTableExportStore} from "~/stores/tableExportStore"
+
+
+const tableExportStore = useTableExportStore()
+const supportedFileFormate: string[] = [
+  "CSV",
+  "JSON",
+]
+
+let selectedItem: DropdownItem = {label: ""}
+const items: DropdownItem[][] = []
+
+supportedFileFormate.forEach((format, index) => {
+  items.push([
+    {
+      label: format,
+      shortcuts: ["" + (index + 1)],
+      disabled: (index == 0),
+      icon: (index == 0) ? "i-heroicons-check-16-solid" : "",
+      class: "opacity-100",
```

Comment:
this causes an type error
I think we can remove it, or do we really need it?
I tried removing it and nothing changed

Reply:
Everything is fine